module-check/src/main/clojure/clojure/core/typed/base_env.clj

```diff
@@ -1445,6 +1445,10 @@ clojure.set/union (All [x] [(Set x) * -> (Set x)])
 clojure.set/intersection (All [x] [(Set x) (Set x) * -> (Set x)])
 clojure.set/difference (All [x] [(Set x) (Set Any) * -> (Set x)])
 
+clojure.repl/apropos [String -> (clojure.lang.LazySeq String)]
```

Comment:
Ambrose will know better, but isn't sequence is defined as `(Seq String)`, where `Seq` is `clojure.core.typed.Seq`.